test/com/facebook/buck/java/intellij/ProjectIntegrationTest.java

```diff
@@ -443,19 +443,32 @@ public void testBuckProjectWithMultipleLibrariesInOneBuildFile() throws IOExcept
   }
 
   @Test
-  public void testNonexistentTarget() throws IOException {
+  public void testProjectWithColon() throws IOException {
     ProjectWorkspace workspace = TestDataHelper.createProjectWorkspaceForScenario(
         this,
         "project1",
         temporaryFolder);
     workspace.setUp();
 
-    expectedException.expect(HumanReadableException.class);
```

Comment:
I should have said that I fixed it locally; it should go out later today.

Our internal CI roughly does this to verify: `ant clean default lint && ./bin/buck build buck && ./bin/buck test`